src/content/docs/merge-queue/performance.mdx

```diff
@@ -44,13 +44,15 @@ Based on this trade-off, there are 3 scenarios that you can choose from:
 
 - **Reliable and Fast:** If you aim to reduce latency and maximize throughput
   without considering CI costs, enable **[speculative
-  checks](speculative-checks)**. This feature lets Mergify test pull requests
+  checks](/merge-queue/speculative-checks)**. This feature lets 
+  Mergify test pull requests
   in parallel, predicting potential merges and executing CI runs
   simultaneously. Speculative checks can slightly increase CI cost: there might
   be certain scenarios that are tested and whose results won't be used if a
   pull request ahead in the queue fails.
```

Comment:
You should rewrap the whole paragraph here too. You should be able to do it with your IDE.

Reply:
Done